### PR TITLE
Updates to Export Service to meet detailed requirements

### PR DESCRIPTION
**Pull request by pazaan**

### CODE REVIEW

**Comment by pazaan** [comment]
[Backend work to Allow Data Export from Tidepool for Web UI](https://trello.com/c/bQuqmF0x/399-backend-work-to-allow-data-export-from-tidepool-for-web-ui)

**Comment by pazaan** [comment]
[[backend] Convert export data from mmol to mg/dL](https://trello.com/c/wLOkfeOP/687-backend-convert-export-data-from-mmol-to-mg-dl)

**Comment by pazaan** [comment]
[[backend] additional data export requirements](https://trello.com/c/PB0BEaMv/688-backend-additional-data-export-requirements)

**Comment by pazaan** [comment]
[Include Ketones in Export](https://trello.com/c/Z7NLK81j/697-include-ketones-in-export)

File: node-data-tools/test/exporter.test.js [hunk 1]
```diff
@@ -32,31 +37,56 @@ const excludedFields = [
...
   'dataSetType',
   'guid',
-  'clockDriftOffset',
-  'conversionOffset',
-  'timezoneOffset',
 ];
 
 const wb = new Excel.Workbook();
 
 (async () => {
```
Comment (derrickburns) [inline node-data-tools/test/exporter.test.js]: Ahhhh.  Modern constructs!  Yay.  Love async/await

**Pull request merged**